src/wiktextract/datautils.py

```diff
@@ -6,6 +6,7 @@
 from collections import defaultdict
 from functools import lru_cache, partial
 from typing import Any, Dict, Iterable, List, Tuple
+from wiktextract.extractor.es.models import BaseModelWrap
```

Comment:
This line would import `pydantic` and the Spanish extractor code for all other extractors code import this file, this is not ideal.

Reply:
I made some changes in #401, maybe that's enough for removing boilerplate methods in `BaseModelWrap`.

Reply:
That was a great change to remove the need for including the pydantic dependency everywhere.  :+1:

Regarding the boilerplate:
- The `.get()` method is needed as long as we use this in our utility functions such as in [append_base_data](https://github.com/tatuylonen/wiktextract/blob/1ccc547567b872df06019a372d006bc5519d1746/src/wiktextract/datautils.py#L249C8-L249C17) 
- For `__getitem__()`, the same applies, e.g. [here](https://github.com/tatuylonen/wiktextract/blob/1ccc547567b872df06019a372d006bc5519d1746/src/wiktextract/datautils.py#L250).
- The same for `__setitem__()`: [here](https://github.com/tatuylonen/wiktextract/blob/1ccc547567b872df06019a372d006bc5519d1746/src/wiktextract/datautils.py#L258)

- The `update()` function isn't strictly necessary but still useful to have. 

I will updated at least the new code to avoid dict like assignments.

Reply:
The new code using pydantic models shouldn't use `append_base_data()`, you could append the new `WordEntry` directly.

Reply:
Thanks! I updated the code accordingly and indeed could get rid of the boilerplate.

Perhaps the `append_base_data()` function should then be moved to the chinese extractor code?